grafana/dashboards/supercharges.json

```diff
@@ -0,0 +1,1067 @@
+          "refId": "A",
+          "select": [
+            [
+              {
+                "params": [
+                  "value"
+                ],
+                "type": "column"
+              }
+            ]
+          ],
+          "timeColumn": "time",
+          "where": [
+            {
+              "name": "$__timeFilter",
+              "params": [],
+              "type": "macro"
+            }
+          ]
+        }
+      ],
+      "timeFrom": null,
+      "timeShift": null,
+      "title": "Supercharger History",
+      "transform": "table",
+      "type": "table"
+    },
+    {
+      "cacheTimeout": null,
+      "colorBackground": false,
+      "colorValue": false,
+      "colors": [
+        "#299c46",
+        "rgba(237, 129, 40, 0.89)",
+        "#d44a3a"
+      ],
+      "datasource": "TeslaMate",
+      "format": "kwatth",
+      "gauge": {
+        "maxValue": 100,
+        "minValue": 0,
+        "show": false,
+        "thresholdLabels": false,
+        "thresholdMarkers": true
+      },
+      "gridPos": {
+        "h": 2,
+        "w": 8,
+        "x": 0,
+        "y": 20
+      },
+      "id": 10,
+      "interval": null,
+      "links": [],
+      "mappingType": 1,
+      "mappingTypes": [
+        {
+          "name": "value to text",
+          "value": 1
+        },
+        {
+          "name": "range to text",
+          "value": 2
+        }
+      ],
+      "maxDataPoints": 100,
+      "nullPointMode": "connected",
+      "nullText": null,
+      "postfix": "",
+      "postfixFontSize": "70%",
+      "prefix": "Energy added:",
+      "prefixFontSize": "50%",
+      "rangeMaps": [
+        {
+          "from": "null",
+          "text": "N/A",
+          "to": "null"
+        }
+      ],
+      "scopedVars": {
+        "car_id": {
+          "selected": false,
+          "value": "1"
+        }
+      },
+      "sparkline": {
+        "fillColor": "rgba(242, 204, 12, 0.25)",
+        "full": false,
+        "lineColor": "#F2CC0C",
+        "show": false,
+        "ymax": null,
+        "ymin": null
+      },
+      "tableColumn": "",
+      "targets": [
+        {
+          "format": "time_series",
+          "group": [],
+          "metricColumn": "none",
+          "rawQuery": true,
+          "rawSql": "SELECT start_date as \"time\", charge_energy_added\nFROM charging_processes\nWHERE \n  $__timeFilter(end_date) AND \n  car_id = $car_id AND     \n   (charge_energy_added * 60 / NULLIF (duration_min, 0)) > 20 AND\n  ('${geofence:pipe}' = '-1' OR geofence_id in ($geofence))\nORDER BY start_date;",
```

Comment:
As I see, "supercharges" are determined by the charging speed:

```SQL
(charge_energy_added * 60 / NULLIF (duration_min, 0)) > 20
```

Due to the name of the dashboard I would have expected that only SuperCharges and no charges at other fast chargers would be displayed. If we wanted to include only the former we could filter for charging sessions where `fast_charger_brand` is `Tesla`.

Reply:
As I think about this.... maybe I will think over a ability to use fast and/or super chargers and then set a filter. I'll work on it this weekend. Maybe modifying the charging dashboard is the best route too... I had thought about it too.